docs/source/developer/interactives.rst

```diff
@@ -112,32 +49,52 @@ Every interactive will follow a very similar file structure as given below:
       ├── static/
       │   ├── interactives/
       │   │   └── <interactive-slug>/
-      │   │       ├── css/
+      │   │       ├── scss/
       │   │       │   ├── third-party/
-      │   │       │   └── style.css
+      │   │       │   └── <interactive-slug>.scss
       │   │       ├── img/
       │   │       ├── js/
       │   │       │   ├── third-party/
-      │   │       │   └── script.js
-      │   │       ├── README.md
-      │   │       └── thumbnail.png
+      │   │       │   └── <interactive-slug>.js
+      │   │       ├── package.json
+      │   │       └── README.md
       │   ├── js/
       │   │   └── third-party/
-      │   └── css/
+      │   └── scss/
       │       └── third-party/
       └── templates/
           └── interactives/
-              └── <interactive-template>.html
+              └── <interactive-slug>.html
 
 .. note::
 
   Each interactive has a unique ``slug``, which is used in file/directory names and urls. For example the slug for the ``High Score Boxes`` interactive is ``high-score-boxes``.
```

Comment:
This contradicts my keys thing from earlier, now I'm not so sure

Reply:
I'm confused on which one you prefer now

Reply:
I now think it's fine to call them slugs, but only in the dev docs